buildenv/jenkins/testJobTemplate

```diff
@@ -326,8 +326,20 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
 							choiceParam('TEST_FLAG', ['', 'JITAAS', 'AOT'], "Optional. Only set to JITAAS/AOT for JITAAS/AOT feature testing.")
-							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the test command")
-							stringParam('JVM_OPTIONS', "", "Use this to replace the test original command line options")
+							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS,'''
+							Use this to append options to the commandline. In general, JVM options can be directly used. For system tests, the special characters may need to be escaped in EXTRA_OPTIONS and JVM_OPTIONS.<br> 
+                            Example 1:<br>
+                            '-Xjit:count=0,\{java/lang/reflect/Method.getParameterAnnotations*\}\(traceFull,log=getParameterAnnotations.log\)'
+                            Example 2:<br>
+                            -Xjit:\"{java/util/HashMap.*}(optlevel=noopt)\"
+							''')
+							stringParam('JVM_OPTIONS', "", '''
+							Use this to append options to the commandline. In general, JVM options can be directly used. For system tests, the special characters may need to be escaped in EXTRA_OPTIONS and JVM_OPTIONS.<br> 
```

Comment:
For JVM_OPTIONS, please keep the original description as the first sentence. That is, change `Use this to append options to the commandline` -> `Use this to replace the test original command line options. In general, JVM options can be directly used...`. The rest description and example can be the same as EXTRA_OPTIONS

Reply:
Done. Once you approve the change I will squash the commits.